confirmation/models.py

```diff
@@ -52,14 +52,16 @@ def generate_key():
 
 def get_object_from_key(confirmation_key):
     # type: (str) -> Union[bool, PreregistrationUser, EmailChangeStatus]
+    if len(confirmation_key) != 24:
```

Comment:
it can be 24 or 40. Before this line, we can add a comment like `# confirmation keys used to be 40 characters`